src/app/_components/Event/Card.js

```diff
@@ -0,0 +1,123 @@
+'use client';
+
+import CalendarIcon from '@icons/calendarFilled.svg';
+import PriceIcon from '@icons/priceFilled.svg';
+import LocationIcon from '@icons/locationFilled.svg';
+import TimeIcon from '@icons/timeFilled.svg';
+import PropTypes from 'prop-types';
+import { Label } from '../Label';
+import { cn } from '@/utils/cn';
+import { OverflownText } from '../OverflownText';
+
+const months = {
+  0: 'січня',
+  1: 'лютого',
+  2: 'березня',
+  3: 'квітня',
+  4: 'травня',
+  5: 'червня',
+  6: 'липня',
+  7: 'серпня',
+  8: 'вересня',
+  9: 'жовтня',
+  10: 'листопада',
+  11: 'грудня',
+};
+
+const weekDays = {
+  0: 'неділя',
+  1: 'понеділок',
+  2: 'вівторок',
+  3: 'середа',
+  4: 'четвер',
+  5: "п'ятниця",
+  6: 'субота',
+};
+
+function ListItem({ icon, text, textColor, fontWeight }) {
+  return (
+    <li className="flex gap-2">
+      {icon}
+      <p className={cn('w-56 text-p3', textColor, fontWeight)}>{text}</p>
+    </li>
+  );
+}
+
+function transformData(event) {
```

Comment:
🟢  Consider on exporting this function somewhere, so it can be reusable for EventsPage.

Reply:
Can you, please, tell some use cases for this function on events page or somewhere else? Personally I think that it will be used only to render cards properly, but i may be wrong

Reply:
On EventsPage we have filters by month.

Reply:
Moved months to lib/consts